src/app/cases/creator/case-creator-submit.component.ts

```diff
@@ -44,6 +47,29 @@ export class CaseCreatorSubmitComponent implements OnInit {
     return (sanitizedEditForm: CaseEventData) => this.casesService.validateCase(this.jurisdictionId, this.caseTypeId, sanitizedEditForm);
   }
 
+  private canSaveDraft(): boolean {
+    return this.eventTrigger.can_save_draft === true;
+  }
+
+  private isCreatingDraft(): boolean {
+    return !this.eventTrigger.draft_store_id;
```

Comment:
can we rename draft_store_id to draft_id?

Reply:
ok